test/ad_backends.jl

```diff
@@ -22,7 +22,7 @@
     # end
     @timed_testset "ReverseDiff" begin
         using ReverseDiff
-        chn = sample(m, NUTS(; adtype=AutoReverseDiff(; compile=true)), 2)
+        chn = sample(m, NUTS(; adtype=AutoReverseDiff(true)), 2)
```

Comment:
@yebai are you sure about this?
I've followed `ADTypes.jl` test case for this:

https://github.com/SciML/ADTypes.jl/blob/main/test/runtests.jl#L39

Reply:
Ok, I also learned something new here :)